packages/pwa/src/components/feedItems/YouTubeFeedItem.tsx

```
@@ -2,16 +2,56 @@
 
 import type {YouTubeFeedItem} from '@shared/types/feedItems.types';
 
-import {FeedItemHeader, FeedItemWrapper} from '@src/components/feedItems/FeedItem';
-import {FeedItemMarkdown} from '@src/components/feedItems/FeedItemMarkdown';
+import {useYouTubeFeedItemTranscript} from '@sharedClient/services/feedItems.client';
+
+import {Text} from '@src/components/atoms/Text';
+import {ImportingFeedItem} from '@src/components/feedItems/ImportingFeedItem';
+import {Markdown} from '@src/components/Markdown';
+
+import {FeedItemHeader, FeedItemWrapper} from './FeedItem';
```

ESLint

[error] './FeedItem' import is restricted from being used by a pattern. Use imports like `@src` and `@shared` instead of relative paths.

(no-restricted-imports)